Task_DEV_4/Task_DEV_4/Triangle.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+
+namespace Task_DEV_4
+{
+    class Triangle : Figure
+    {
+        private const string ERROR_MESSAGE = "Side cannot be less equals than 0.";
+        private const string ERROR_MESSAGE2 = "The sum of the two sides cannot be less than the third side.";
```

Comment:
please, give more clarity names for constants.
code style

Reply:
Done.